### ISSUE TITLE
Two Columns Layouts not work, when I open any document and one extension readme at a time

### ISSUE THREAD

**Issue by JoseChirinos** [microsoft/vscode#80296]
1. Open any document
2. Open any extension (see readme)
3. Click en [|] button for see Two Columns Layouts

Result: all tabs are hidden

**Comment by sbatten** [comment]
I could repro in stable though after reproing once, it didn't happen 100% of the time.

Console error
```
bootstrap-window.js:215 Error: Invalid guestInstanceId: null
    at Object.getGuestForWebContents (C:\Users\stbatt\AppData\Local\Programs\Microsoft VS Code\resources\electron.asar\browser\guest-view-manager.js:425:11)
    at C:\Users\stbatt\AppData\Local\Programs\Microsoft VS Code\resources\electron.asar\browser\rpc-server.js:449:34
    at EventEmitter.ipcMain.on.args (C:\Users\stbatt\AppData\Local\Programs\Microsoft VS Code\resources\electron.asar\browser\rpc-server.js:278:21)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (C:\Users\stbatt\AppData\Local\Programs\Microsoft VS Code\resources\electron.asar\browser\api\web-contents.js:370:21)
    at WebContents.emit (events.js:182:13)
```

**Comment by deepak1556** [comment]
This happens 100% on all three flavors of VSCode, with the first run when opening an extension preview on clean editor state and trying `Split Editor`. It seems like webview is detached and reattached by vscode during this action while some pending events are fired which results in this error.

Demo: https://streamable.com/asnyf

@mjbvz thoughts ?

**Comment by bpasero** [comment]
It is great to have a repro for this as it spams our error telemetry with lots of errors and we have not been able to figure this out yet. See https://github.com/microsoft/vscode/issues/79165

**Comment by mjbvz** [comment]
Fixed by 92dac9e20e193d85d732428bff455349feb723d1

Split was causing the webview in the extension editor to be reparented, which caused this cryptic error
